procrastinate/retry.py

```diff
@@ -85,6 +171,7 @@ class RetryStrategy(BaseRetryStrategy):
     retry_exceptions: Iterable[type[Exception]] | None = None
 
     def get_schedule_in(self, *, exception: BaseException, attempts: int) -> int | None:
+        # TODO: remove this method in a future version and move logic to `get_retry_decision`
```

Comment:
Is there any benefit in not doing it now ? Users aren't going to call get_schedule_in themselves, are they ?

Reply:
Probably not. I removed it.